doc/ref_cert/lfn/chapters/chapter02.md

```diff
@@ -14,38 +14,58 @@
 
 <a name="2.1"></a>
 ## 2.1 Introduction
-
-Describe the purpose.. 
-
-- Define E2E Framework Requirements, process and automation for NFVI Certification
-- Ensure Platform Stability, Operability, & Performance for target RAs
+NFVI (Network Functions Virtualization Infrastructure) refers to the physical resources (compute, storage and network) on which virtual network functions (VNFs) are deployed. As such, the performance of a VNF depends on the underlying NFVI over which it is hosted. A certain VNF may perform good in one hardware and may perform worst in another. Thus, a need arises to certify NFVI that can help in onboarding VNFs onto a hardware with an acceptable level of VNF performance. Certain frameworks like [yardstick](https://github.com/opnfv/yardstick), [vsperf](https://github.com/opnfv/vswitchperf) etc. provide a set of tests that can be run on a hardware to obtain Key Performance Indicators (KPIs) which give a measurable output of the NFVI's performance. With these KPIs, a decision can be made on the VNFs that can offer an acceptable level of performance when on-boarded on the NFVI.
 
 <a name="2.2"></a>
 ## 2.2 Methodology
-
-Define the approach to NFVI testing, and Frameworks needed...
+The NFVI provided by hardware vendors is consumed or used by VNFs via APIs exposed by Virtualised Infrastructure Manager (VIM). The resources created by VIM on the NFVI use the underlying physical hardware (compute, storage and network) either directly or indirectly. CNTT recommends RA1/RA2 to be used as a reference architecture for NFVI certification. This  would provide a set of standard interaces to create resources on NFVI. Below step by step process illustrates the NFVI certification methodology:
+* CNTT RA1/RA2 is deployed on NFVI that needs to be certified.
+* A set of tests are run on NFVI to determine the NFVI readiness for certification process,
+* Golden KPIs are taken as a reference.
+* A set of tests are run on the target NFVI.
+* KPIs obtained from the target NFVI are collected and submitted to certification portal.
+* The NFVI KPIs are reviewed and compared with Golden KPIs to determine if the certification badge is to be provided to NFVI or not.
 
 <a name="2.3"></a>
 ## 2.3 Certification Strategy & Vehicle
-
-Define and illustrate the certification strategy and vehicle for NFVI..
+In order to begin the certification process, NFVI needs to be validated and expected to be in a required state. This state would be determined by running tests as described in Reference Implementation. Once the target NFVI passes these tests, it would become a candidate for NFVI certification. If NFVI fails the tests, it will not be move to next workflow for certification.
 
```

Comment:
Add --

_**NFVI+VNF validations consist of a three part process for Compliance, Validation, and Performance.**  Adherence to **Security** standards are equally important and addressed in Chapter [where].

The three part verificaiton process includes **NFVI Manifest Validations, Emprical Baseline measurements against targeted VNF families**, and **Candidate VNF verifications**.  More specifically,

- **NFVI Verification** _(Compliance)_: NFVI is the SUT, ensuring NFVI is compliant with specs of Ref Model and Ref Architecture accomplished with **Manifest Validations** (performed via Echo Tests) 

- **Empirical Validation with Reference VNF** _(Validation)_: NFVI is the SUT, ensuring NFVI runs with Golden VNFs and is instrumented to objectively validate resources through consumption and measurement

- **Candidate VNF Certification** _(Validation & Performance)_: VNF is the SUT, ensuring VNFs operate with Ref Model and Ref Arch leveraging VVP/CVP/VFNSDK Test Suites

- **Security**: Ensures NFVI+VNF is free from known security vulnerabilities, utilizing industry standard cyber security frameworks (Refer to CNTT Chapter XX Security for additional test/verification details)

Validations are performed against an **Infrastructure Profile Catalog, VNF performance profile**, and **targeted VNF class** or family for baseline measurements.

The **Infrastructure Profile Catalog** contains the following attributes: 

- Profile is a collection of (limited) options offered by the infrastructure to the VNF
  - Capabilities
  - Metrics
  - Compute flavours
  - Interface options
  - Storage extensions
  - Acceleration capabilities

- Profiles are offered to VNFs as an instance types with predefined compute flavours (T-shirt size)
  - A particular set of options is an instance type
  - Compute flavours: S, M, L

**VNF performance profiles**, for which NFVI validations will support and be verified against, are defined as basic, network intensive, and compute intensive. Details for each of these profiles can be found in chapter 2.3._

![infra_profiles](https://user-images.githubusercontent.com/5643130/71145555-ac4b4f00-21e7-11ea-8d1d-3473fb33fb2d.png)

**Targeted VNF Classes/Families** for baseline measurements are described in chapter XXXX.

Reply:
Added content (Todo formatting)

Reply:
Let me know if we can resolve this.

Reply:
@msidana - Manik, I could **not** see the image.  Did you upload the image to the figure directory?  CNTT/doc/ref_cert/lfn/figures/ .  Besides the image, the text looks good.

Reply:
@mf4716 Yes, even I can't see them. All figures are uploaded to ```doc/ref_cert/lfn/figures/``` directory and are showing up in "Files Changed" tab of this PR.

Reply:
Okay.  Let's close/resolve and address any issues after merge.  I'm fairly confident it will work once the images are successfully updated (merged) in the Figures directory.  Resolving this thread.